### PR TITLE
Added Device and Operation Related Platform Tests for Oracle Platform

### PR DESCRIPTION
**Pull request by dilee**

### CODE REVIEW

File: modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/mobileDevice/OracleMobileDeviceManagement.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import junit.framework.Assert;
+import org.apache.commons.httpclient.HttpStatus;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.annotations.ExecutionEnvironment;
+import org.wso2.carbon.automation.engine.annotations.SetEnvironment;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.mdm.integration.common.*;
```
Comment (harshanL) [inline modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/mobileDevice/OracleMobileDeviceManagement.java:31]: Do not use .* imports. Please fix it.

File: modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/mobileDevice/OracleMobileDeviceManagement.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    @Test(dependsOnMethods = {"addEnrollment"}, description = "Test view device types")
+    public void testViewDeviceTypes() throws Exception {
+        MDMResponse response = client.get(Constants.MobileDeviceManagement.VIEW_DEVICE_TYPES_ENDPOINT);
+        Assert.assertEquals(HttpStatus.SC_OK, response.getStatus());
+        Assert.assertEquals(PayloadGenerator.getJsonPayloadToString
+                (Constants.MobileDeviceManagement.VIEW_DEVICE_RESPONSE_PAYLOAD_FILE_NAME),response.getBody());
+        //Response has two device types, because in windows enrollment a windows device is previously enrolled.
+    }
+
+    //Pagination testings for GetAllDevice Function
```
Comment (prabushi) [inline modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/mobileDevice/OracleMobileDeviceManagement.java:104]: Correct method level java docs. And, please add doc comments for other test methods too.

Comment (dilee) [reply modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/mobileDevice/OracleMobileDeviceManagement.java:104]: Fixed in https://github.com/wso2-attic/product-emm/pull/389

File: modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/mobileDevice/OracleMobileDeviceManagement.java (hunk 1 above)
Comment (dilee) [reply modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/mobileDevice/OracleMobileDeviceManagement.java:31]: Fixed in https://github.com/wso2-attic/product-emm/pull/389

**Pull request merged**